src/main/java/betterquesting/client/QuestNotification.java

```diff
@@ -76,19 +76,24 @@ public void onDrawScreen(RenderGameOverlayEvent.Post event) {
             alpha = MathHelper.clamp_float(alpha, 0.02F, 1F);
             final int color = new Color(1F, 1F, 1F, alpha).getRGB();
 
+            GL11.glColor4f(1F, 1F, 1F, alpha);
+
             if (notice.icon != null && alpha > 0.2F) {
                 RenderUtils.RenderItemStack(mc, notice.icon, width / 2 - 8, height / 4 - 20, "", color);
             }
 
-            String tmp = EnumChatFormatting.UNDERLINE + "" + EnumChatFormatting.BOLD + QuestTranslation.translate(notice.mainTxt);
-            int txtW = RenderUtils.getStringWidth(tmp, mc.fontRenderer);
-            mc.fontRenderer.drawString(tmp, width / 2 - txtW / 2, height / 4, color, true);
+            GL11.glEnable(GL11.GL_BLEND);
```

Comment:
this in not needed since both are already enabled by vanilla code before calling the event